SQL/tgstation_schema.sql

```diff
@@ -254,6 +254,21 @@ CREATE TABLE `messages` (
 /*!40101 SET character_set_client = @saved_cs_client */;
 
 --
+-- Table structure for table `role_time`
+--
+
+DROP TABLE IF EXISTS `role_time`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+
+CREATE TABLE `role_time` 
+( `ckey` VARCHAR(32) NOT NULL ,
+ `job` VARCHAR(128) NOT NULL ,
+ `minutes` INT UNSIGNED NOT NULL,

```

Comment:
So I did the math on this, (because it's my job).

In 8,378 years, we will have to upgrade this to a long.

Think we can manage that by then?